src/libraries/System.Runtime/tests/System/Reflection/InvokeRefReturn.cs

```diff
@@ -39,6 +39,7 @@ public static void TestRefReturnNullableNoValue()
         [Theory]
         [MemberData(nameof(RefReturnInvokeTestData))]
         [ActiveIssue("https://github.com/dotnet/runtime/issues/67457", TestRuntimes.Mono)]
+        [ActiveIssue("https://github.com/dotnet/runtime/issues/69755", typeof(PlatformDetection), nameof(PlatformDetection.IsNativeAot))]
```

Comment:
I assume you have this fix #69440 since it addresses similar issue

Reply:
Yes, I have that fix and the test was still failing